### PR TITLE
Palindrome exercise

### PR DESCRIPTION
**Pull request by anciuvaga**

### CODE REVIEW

File: src/javaExercises/Palindrome/Palindrome.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        }
+        return reversedString.toString().equals(cleanString);
+    }
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input the desired word:");
+        boolean palindromeCheck = verifyIfPalindrome(sc.nextLine());
+        if (palindromeCheck == true) {
```
Comment (CristianManic) [inline src/javaExercises/Palindrome/Palindrome.java]: why do you need this check? **if()** statement needs a boolean value inside it, as long as ***palindromeCheck*** is a boolean, why do you need an operator that return an boolean value to check with another value. It's confusing even when explaining, try to analyze what your code is performing

File: src/javaExercises/Palindrome/Palindrome.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            if (charBackward != charForward)
+                return false;
+        }
+        return true;
+    }
+
+    public static boolean verifyIfPalindrome(String text) {
+
+        StringBuilder reversedString = new StringBuilder();
+        String cleanString = text.replaceAll("\\s+", "");
```
Comment (CristianManic) [inline src/javaExercises/Palindrome/Palindrome.java]: there's a `trim()` method for this, don't get complicated

File: src/javaExercises/Palindrome/Palindrome.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+package javaExercises.Palindrome;
+
+import java.util.Scanner;
+
+public class Palindrome {
+    public static boolean checkIfPalindrome(String text) {
+        String cleanString = text.replaceAll("\\s+", "");
```
Comment (CristianManic) [inline src/javaExercises/Palindrome/Palindrome.java]: same for this

File: src/javaExercises/Palindrome/Palindrome.java (hunk 1 above)
Comment (anciuvaga) [reply src/javaExercises/Palindrome/Palindrome.java]: Done

File: src/javaExercises/Palindrome/Palindrome.java (hunk 2 above)
Comment (anciuvaga) [reply src/javaExercises/Palindrome/Palindrome.java]: Done

File: src/javaExercises/Palindrome/Palindrome.java (hunk 3 above)
Comment (anciuvaga) [reply src/javaExercises/Palindrome/Palindrome.java]: Done

**Pull request merged**